datasette/templates/__sorting_example.html

```diff
@@ -0,0 +1,29 @@
+<script src="{{ urls.static('sort_utils.js') }}"></script>
+<div>
+    <p style="display:inline-block;">Sorting options:</p>
+    <button type="button" class="btn-secondary" id="toggle_sort_menu" onclick="toggleSortMenu();" style="display:inline-block;">show</button>
+</div>
+<div style="display:block;position: absolute;z-index: 1;">
+    <div class="dropdown-menu" id="sort_menu" style="display:none;">
+        <div class="hook"></div>
+        <div class="row" name="sorting_options">
+        <div class="col">
+            <p style="margin:0;">Column | Type</p>
+        </div>
+        {% for column in display_columns if column.sortable %}
+            <div class="col">
+                <input type="checkbox" name="{{column.name}}">
+                {{column.name}} 
+                <input type="radio" id="{{column.name}}_asc" name="{{column.name}}_t" value="asc" checked>
```

Comment:
Why there's a `_t` suffix in the name?

Reply:
I added the suffix in the radiobuttons names to differentiate them from their checkboxes ("_t" = type of sorting - ascending/descending).